armotypes/attackchainstypes.go

```diff
@@ -15,6 +15,11 @@ const (
 )
 
 type AttackChain struct {
+	AttackChainNodes AttackChainNode `json:"attackChainNodes,omitempty" bson:"attackChainNodes,omitempty"`
+	AttackChainConfig
```

Comment:
Don't we need `json:",inline" bson:",inline"` here?

Reply:
bson -> we will not use AttackChain struct to save it in mongo, only AttackChainConfig so i think it is not needed
json -> i tested it [here](https://go.dev/play/p/vDej_AX_4hr) and seems it is not needed